stable_baselines3/common/vec_env/vec_monitor.py

```diff
@@ -23,9 +23,6 @@ class VecMonitor(VecEnvWrapper):
     :param info_keywords: extra information to log, from the information return of env.step()
     """
 
-    episode_returns: np.ndarray
```

Comment:
I usually do that to declare explicitly members that are not defined in the `__init__`.
The goal is that the user knows easily what are the members of a given class by looking only at the `__init__`.
Or is there a better way?

Reply:
I see, I thought it was only to satisfy mypy.
I don't know a better way. I think you have to look at it on a case-by-case basis. Here for example, we could simply add it in the init:

```python
class VecMonitor(VecEnvWrapper):
     def __init__(self, ...):
        ...
        self.episode_returns = np.zeros(self.num_envs, dtype=np.float32)
        self.episode_lengths = np.zeros(self.num_envs, dtype=np.int32)
```

or keep what you had originally done. As you like!